CODING_STANDARD.md

```diff
@@ -333,18 +333,21 @@ the [LLVM Snapshot Builds page](http://llvm.org/builds/).
 
 ### FORMATTING A RANGE OF COMMITS
 
-Clang-format is distributed with a driver script called git-clang-format-3.8.
+Clang-format is distributed with a driver script called git-clang-format-7.
 This script can be used to format git diffs (rather than entire files).
 
 After committing some code, it is recommended to run:
 ```
-git-clang-format-3.8 upstream/develop
+git-clang-format-7 upstream/develop
 ```
 *Important:* If your branch is based on a branch other than `upstream/develop`,
 use the name or checksum of that branch instead. It is strongly recommended to
 rebase your work onto the tip of the branch it's based on before running
 `git-clang-format` in this way.
 
+Note: git-clang-format uses the git config variable `clangformat.binary` so if you 
+have multiple versions of clang-format installed, you might need to update this.
```

Comment:
"or explicitly specify the binary to use via `--binary clang-format-7`."

Reply:
I didn't know that!

Reply:
I didn't know about the `clangformat.binary` variable... ;-)